src/main/java/org/deepsymmetry/beatlink/VirtualCdj.java

```diff
@@ -1451,6 +1429,18 @@ private void deliverDeviceUpdate(final DeviceUpdate update) {
         }
     }
 
+    // This is a way to get mediaDetails from Rekordbox and pass to all MediaDetailsListeners
```

Comment:
It doesn’t come from rekordbox, it is a protocol message supported by true Pro DJ Link hardware.

Reply:
I’m not sure I like using the `MediaDetailsListener` interface here because it is a strange Opus-specific hack we are doing, I’d rather have a package-private method `proxyMediaDetails(MediaDetails details)` that `VirtualRekordbox` can call, which simply calls `deliverMediaDetailsUpdate(details). And actually, we don’t even need a separate method; we can make `deliverMediaDetails()` package-private, and `VirtualRekordbox` can call it directly.

Trying to abstract away the relationship between these two classes is counterproductive, I think. They are necessarily entwined because we are adding support for new hardware that doesn’t really work in the model that the classes were designed to operate with. And making that clear in the code is best, I feel.

Reply:
I adjusted to `This is a way to get mediaDetails from Rekordbox and pass to all MediaDetailsListeners`.

Reply:
Just saw the second comment here. Adjusting.